modules/tinymce/src/plugins/searchreplace/test/ts/browser/SearchReplaceDialogTest.ts

```diff
@@ -129,8 +129,8 @@ describe('browser.tinymce.plugins.searchreplace.SearchReplaceDialogTest', () =>
   it('TINY-9704: Should reset the "Not Found" alert after updating the search preferences', async () => {
     const editor = hook.editor();
     editor.setContent('<p>tiny tiny tiny tiny</p>');
-    TinySelections.setSelection(editor, [ 0, 0 ], 5, [ 0, 0 ], 9);
     await Utils.pOpenDialog(editor);
+    await Utils.pSetFieldValue(editor, 'input.tox-textfield[placeholder="Find"]', 'tiny');
```

Comment:
Is this right setting a value then immediately asserting that same value the other tests in the file seems to use the setSelection to select a word or doesn't it matter for this case?

Reply:
Selection does not matter for this case I believe. The issue is with the error alert and the default selection within the other tests is just a way to fill the input by default, I believe.

Assertion is redundant indeed though.